src/content/en/updates/2017/11/dynamic-import.md

```diff
@@ -0,0 +1,151 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Dynamic import() introduces a new function-like form of import that unlocks new capabilities compared to static import. This article compares the two and gives an overview of what’s new.
+
+{# wf_updated_on: 2017-11-20 #}
+{# wf_published_on: 2017-11-20 #}
+{# wf_tags: javascript #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: Dynamic `import()` introduces a new function-like form of `import` that unlocks new capabilities compared to static `import`. This article compares the two and gives an overview of what’s new. #}
+{# wf_blink_components: Blink>JavaScript>Language #}
+
+# Dynamic import() {: .page-title }
+
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+
+Note: Dynamic `import()` is available in Chrome 63 and [Safari Technology Preview
+24](https://webkit.org/blog/7423/release-notes-for-safari-technology-preview-24/).
+
+[Dynamic `import()`](https://github.com/tc39/proposal-dynamic-import) introduces a new
+function-like form of `import` that unlocks new capabilities compared to static `import`. This
+article compares the two and gives an overview of what’s new.
+
+## Static `import` (recap)
+
+Back in September, Chrome 61 shipped with support for the ES2015 `import` statement within
+[modules](https://jakearchibald.com/2017/es-modules-in-browsers/).
+
+Consider the following module, located at `./utils.js`:
+
+```js
+// Default export
+export default () => {
+  console.log('Hi from the default export!');
+};
+
+// Named export `doStuff`
+export const doStuff = () => {
+  console.log('Doing stuff…');
+};
+```
+
+Here’s how to statically import and use the `./utils.js` module:
+
+```html
+<script type="module">
+  import * as module from './utils.js';
+  module.default();
+  // → logs 'Hi from the default export!'
+  module.doStuff();
+  // → logs 'Doing stuff…'
+</script>
+```
+
+This syntactic form for importing modules is a **static** declaration: it only accepts a string
+literal as the module specifier, and introduces bindings into the local scope via a pre-runtime
+“linking” process. The static `import` syntax can only be used at the very start of the file.
+Static `import` enables important use cases such as static analysis, bundling tools, and
+tree-shaking.
+
+In some cases, it’s useful to:
+
+* import a module on-demand (or conditionally)
+* compute the module specifier at runtime
+* import a module from within a regular script (as opposed to a module)
+
+None of those are possible with static `import`.
+
+## Dynamic `import()` 🔥
+
+[Dynamic `import()`](https://github.com/tc39/proposal-dynamic-import) introduces a new
+function-like form of `import` that caters to those use cases. `import(moduleSpecifier)` returns a
+promise for the module namespace object of the requested module, which is created after fetching,
```

Comment:
promise -> Promise

Reply:
Lowercase is consistent with https://developers.google.com/web/fundamentals/primers/async-functionsand https://developers.google.com/web/fundamentals/primers/promises and with what you’d say for e.g. strings (“return a string”, not “return a `String`”)